docs/tutorial/debugging-main-process-vscode.md

```diff
@@ -35,3 +39,49 @@ $ code electron-quick-start
 Set some breakpoints in `main.js`, and start debugging in the [Debug View](https://code.visualstudio.com/docs/editor/debugging). You should be able to hit the breakpoints.
 
 Here is a pre-configured project that you can download and directly debug in VSCode: https://github.com/octref/vscode-electron-debug/tree/master/electron-quick-start
+
+## Native C++ (Windows)
+
+### 1. Open an Electron project in VSCode.
+
+```sh
+$ git clone git@github.com:electron/electron-quick-start.git
+$ code electron-quick-start
+```
+
+### 2. Add a file `.vscode/launch.json` with the following configuration:
+
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "(Windows) Launch",
+      "type": "cppvsdbg",
+      "request": "launch",
+      "program": "${workspaceFolder}\\out\\Testing\\electron.exe",
```

Comment:
Should there be a note that this folder is not necessarily called Testing? I know this name is configurable as per the instructions [in our docs](https://www.electronjs.org/docs/development/build-instructions-gn#_fphzsbpi:~:text=You%20can%20replace%20Testing%20with%20another%20name), and maybe in `build-tools` (forgot if this is the case).

Reply:
Oh, I was not aware of this! Thank you for bringing it up. I could not find any mention of this in the build-tools repository, but @codebytere may know?

Reply:
I've amended this in the next commit by providing a few options for what this could be depending on your build.